packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.spec.ts

```diff
@@ -47,7 +48,7 @@ describe('RichTextEditor', () => {
     it('should initialize Tiptap editor', () => {
         expect(pageObject.editorSectionHasChildNodes()).toBeTrue();
         expect(pageObject.getEditorSectionFirstElementChildClassName()).toBe(
```

Comment:
minor: This should have been caught earlier. Can we remove any html structure details on the function name? As we will anyway update the page object function on updating the html structure or tip tap structure? 

```suggestion
        expect(pageObject.getEditableAreaClassName()).toBe(
```